sdk/maps/azure-maps-examples/src/main/java/com/azure/maps/examples/SearchSample.java

```diff
@@ -158,18 +158,18 @@ public static void main(String[] args) throws IOException {
 
         // simple
         MapsCommon.print(client.fuzzySearch(new FuzzySearchOptions("starbucks")));
-*/
+
         // with options
         SearchAddressResult results = client.fuzzySearch(
             new FuzzySearchOptions("1 Microsoft Way", new LatLong(40.706270, -74.011454))
                 .setTop(5));
-    //    MapsCommon.print(results);
+       MapsCommon.print(results);
 
         // with response
         Response<SearchAddressResult> response = client.fuzzySearchWithResponse(
             new FuzzySearchOptions("Monaco").setEntityType(GeographicEntityType.COUNTRY)
                 .setTop(5), null);
-    //    MapsCommon.print(response.getStatusCode());
+       MapsCommon.print(response.getStatusCode());
```

Comment:
MapsCommon.print() is misaligned/indented. This will cause problems with Checkstyle when we enabled it later.